.style.yapf

```diff
@@ -15,7 +15,7 @@ SPACES_BEFORE_COMMENT=2
 SPACE_BETWEEN_ENDING_COMMA_AND_CLOSING_BRACKET=False
 SPLIT_ARGUMENTS_WHEN_COMMA_TERMINATED=False
 SPLIT_BEFORE_BITWISE_OPERATOR=True
-SPLIT_BEFORE_FIRST_ARGUMENT=True
+SPLIT_BEFORE_FIRST_ARGUMENT=False
```

Comment:
Google style guide allows for both. We are not following the [Google Python style guide](https://google.github.io/styleguide/pyguide.html) to the letter.